src/handlers/summary.ts

```diff
@@ -0,0 +1,28 @@
+import * as core from "@actions/core";
+import { Context } from "../types";
+
+export interface ResultInfo {
+  url: string;
+  merged: boolean;
+}
+
+function generateGitHubSummary(context: Context, urls: ResultInfo[]): string {
+  let monitored = `Monitored: [${context.config.repos.monitor.join(" | ")}]\n\n`;
+  monitored += `Ignored: [${context.config.repos.ignore.join(" | ")}]\n\n`;
+  monitored += urls
+    .map(({ url, merged }) => {
+      const status = merged ? `<span>🟢 merged</span>` : `<span>🔵 no change</span>`;
+      return `- [${url}](${url}) - ${status}`;
+    })
+    .join("\n");
```

Comment:
You should switch the order of the "columns" because the URLs are of variable length, where the indicators there are only two lengths. It will present more cleanly with the indicators first, and then the URLs after.

Reply:
maybe it would be better to just do a table?

Reply:
@0x4007 How about this
https://github.com/Meniole/automated-merging/actions/runs/10501574194/attempts/1#summary-29091781547